.github/workflows/lint.yaml

```diff
@@ -1,6 +1,10 @@
 name: Lint
 
-on: [push, pull_request]
+on:
+  push:
+    branches: master
```

Comment:
OK. Do you know of a way that avoids duplicate jobs for PRs from branches in the same repo? This might become annoying with the 2000 CI minute / month limit per Github org for more active orgs. So I tend to try and avoid dupes wherever possible.